### DISCUSSION TITLE
Being citeable

### DISCUSSION THREAD

**Discussion by Photonsneverrest** [rafael-fuente/Diffraction-Simulations--Angular-Spectrum-Method#11] [Ideas]
Hi Rafael,
could you push a .cff file (https://citation-file-format.github.io/)  so you can be cited and acknowledged for your great work?
https://twitter.com/natfriedman/status/1420122675813441540

**Comment by rafael-fuente** [comment]
Thank you.
Yes, I'll add a Zenodo badge. 
I'm curious. What are you working on?

**Reply by Photonsneverrest** [reply]
I'm working on metasurfaces as part of my master thesis. So i am using your code to model my pattern as grating. I guess the diffraction simulation won't be sufficient/completely realistic but i should rather use a finite-difference time-domain (FDTD) calculation. 
Well for the beginning it is good to have. 
Even so I would be very interested in collaborating, I don't think i can because I am supposed to make the thesis alone :/ (?)

I'm also curious whether you are a student or what is your profession?

**Reply by rafael-fuente** [reply]
I'm a student, I'm also working on a project modeling metasurfaces. 

If the scalar theory is enough, or you require to use FDTD, depends on what you exactly need to simulate.

It's supposed you need to solve the problems of your thesis alone, so I'm not going to help you with that. 
But we can talk/share public scientific biography and ask/discuss the features of this software or another open software we need.

If you want to talk with me, this is my e-mail: rafael.fuente.herrezuelo@gmail.com
If you prefer to talk with me via WhatsApp or a similar platform, this is my public mobile phone: +34 695 152 209

I'm going to release a Zenodo DOI identifier for this software to be citable in literature.

**Comment by Photonsneverrest** [comment]
Yeah that would be cool!
My WhatsApp number: +49 15730957415
I‘m associated with NTNU in Trondheim/Norway. The target of my project is to make an amorphous Silicon based metasurface for the application as polarizer and it shall be accurately characterized. Therefore, I’m mainly going to use a spectroscopic ellipsometer. I am still fabricating first samples in the moment (but they should be finalized by Wednesday). 

Are you mainly simulating or also going to measure something? Maybe there is even something which I could fabricate or measure for you?

A question regarding saving a figure; Am I understanding correctly that I need to supplement the plot_intensity class with a .savefig method? Or is there already a method included to solve a plot?
